client/src/components/Profile/ContactsCard.tsx

```diff
@@ -144,12 +144,28 @@ const ContactsCard = ({ connections, data, isEditingModeEnabled, sendConfirmatio
     setIsSaveDisabled(!readyToUpdate);
   }, [hasError, values, displayValues]);
 
+  const noDataDescriptionWithNote: JSX.Element = (
+    <div style={{ textAlign: 'center' }}>
+      Contacts aren't filled in.
+      <br />
+      <br />
+      <strong>
+        Your contact information will only be visible to course coordinators, RS School admins, and the mentor assigned
+        to you.
+      </strong>
+      <br />
+      Please make sure to provide at least one reliable way to contact you.
+      <br />
+      <strong>Telegram is preferred.</strong>
+    </div>
```

Comment:
Can we use Antd instead of raw HTML tags? E.g. `div` => `Typography.Paragraph`, `strong` => `Typography.Text` with `strong=true` property, etc.